reddwarf/types/agora.py

```diff
@@ -0,0 +1,44 @@
+from typing import TypedDict, List, Optional
+from enum import IntEnum
+
+IncrementingId = int
+
+class Statement(TypedDict):
+     id: IncrementingId
+
+class Participant(TypedDict):
+    id: IncrementingId
+
+class ClusteredParticipant(TypedDict):
+    id: IncrementingId # participant.id
+    x: float
+    y: float
+
+class VoteValueEnum(IntEnum):
+    AGREE = 1
+    DISAGREE = -1
+    # Can withhold using "pass" at own discretion.
+    PASS = 0
```

Comment:
This is my preference now

```suggestion
class VoteValueEnum(IntEnum):
    UP = 1      # agree
    DOWN = -1   # disagree
    # Can withhold using "pass" at own discretion.
    NEUTRAL = 0 # pass
```

Reply:
I don't have preference, entirely up to you!

Reply:
I'm gonna close this for now, as it will make more sense once there's another implementation :)